src/frontends/tensorflow/src/helper_ops/block_lstm.hpp

```diff
@@ -0,0 +1,158 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <string>
+#include <vector>
+
+#include "helper_ops/internal_operation.hpp"
+#include "op_table.hpp"
+#include "openvino/frontend/tensorflow/decoder.hpp"
+#include "tf_framework_node.hpp"
+
+namespace ov {
+namespace frontend {
+namespace tensorflow {
+
+class BlockLSTM : public ov::frontend::tensorflow::InternalOperation {
+public:
+    OPENVINO_OP("BlockLSTM", "ov::frontend::tensorflow::util", ov::frontend::tensorflow::InternalOperation);
+
+    BlockLSTM(const Output<Node>& seq_len_max,
+              const Output<Node>& x,
+              const Output<Node>& cs_prev,
+              const Output<Node>& h_prev,
+              const Output<Node>& w,
+              const Output<Node>& wci,
+              const Output<Node>& wcf,
+              const Output<Node>& wco,
+              const Output<Node>& b,
+              float forget_bias,
+              float cell_clip,
+              bool use_peephole,
+              const std::shared_ptr<DecoderBase>& decoder = nullptr)
+        : ov::frontend::tensorflow::InternalOperation(
+              decoder,
+              OutputVector{seq_len_max, x, cs_prev, h_prev, w, wci, wcf, wco, b},
+              7),
+          m_forget_bias(forget_bias),
+          m_cell_clip(cell_clip),
+          m_use_peephole(use_peephole),
+          m_hidden_size(ov::Dimension::dynamic()),
```

Comment:
You do not initialize `m_batch_size` here. I suggest moving such initialization, to lines 148-150, since it do not depend on arguments anyway.

Reply:
I avoid `batch_size` at all because it must be a part of the sub-graph replacing `BlockLSTM`